```
__________ sympy/integrals/tests/test_heurisch.py:test_heurisch_mixed __________
Traceback (most recent call last):
  File "/home/runner/work/sympy/sympy/sympy/integrals/tests/test_heurisch.py", line 126, in test_heurisch_mixed
    assert heurisch(sin(x/sqrt(-x))) == 2*x*cos(x/sqrt(-x))/sqrt(-x) - 2*sin(x/sqrt(-x))
TypeError: heurisch() missing 1 required positional argument: 'x'
```
You need to pass the symbol argument to heurisch.

--- a/sympy/integrals/tests/test_heurisch.py
+++ b/sympy/integrals/tests/test_heurisch.py
@@ -123,7 +123,7 @@ def test_heurisch_hyperbolic():
 
 def test_heurisch_mixed():
     assert heurisch(sin(x)*exp(x), x) == exp(x)*sin(x)/2 - exp(x)*cos(x)/2
-    assert heurisch(sin(x/sqrt(-x))) == 2*x*cos(x/sqrt(-x))/sqrt(-x) - 2*sin(x/sqrt(-x))
+    assert heurisch(sin(x/sqrt(-x)), x) == 2*x*cos(x/sqrt(-x))/sqrt(-x) - 2*sin(x/sqrt(-x))
 
 
 def test_heurisch_radicals():

--- a/sympy/solvers/ode/tests/test_single.py
+++ b/sympy/solvers/ode/tests/test_single.py
@@ -1259,8 +1259,8 @@ def _get_examples_ode_sol_nth_order_reducible():
 
     'reducible_11': {
         'eq': f(x).diff(x, 2) - f(x).diff(x)**3,
-        'sol': [Eq(f(x), C1 - sqrt(2)*(I*C2 + I*x)*sqrt(1/(C2 + x))),
-        Eq(f(x), C1 + sqrt(2)*(I*C2 + I*x)*sqrt(1/(C2 + x)))],
+        'sol': [Eq(f(x), C1 - sqrt(2)*sqrt(-1/(C2 + x))*(C2 + x)),
+        Eq(f(x), C1 + sqrt(2)*sqrt(-1/(C2 + x))*(C2 + x))],
         'slow': True,
     },
 

